### ISSUE TITLE
Update "Get the Data" section

### ISSUE THREAD

**Issue by katska** [openaustralia/planningalerts#1782]
Update "Get the Data" 

This is the beginning of a big overhaul. First series of changes aim to:
- Introduce two week trial 
- Two week trial users agree to terms of use
- Reframe the offers for commercial and granted use
- Clarify its a paid service first not free by default
- Brings together the terms of use/be nice for clarity
- Community users also subject to an agreement

**Comment by katska** [comment]
the part about non-commercial use being available can included later in the section that's currently labelled usage

**Comment by katska** [comment]
I'm bringing quite a few updates, many changes are coming to "Get the Data to clarify our offerings. Some of these need more work before we can do that. The first step in our timeline is to introduce a 2 week trial, and reframe the broader introduction to what's commercial and what's free.

alrighty this is where I get specific.... 

In the first paragraph, replace:

> 
> Planning application data is available programmatically. Details of the API are listed below.
> 
> Non–commercial, low–volume use of this service is free. [Read more.](https://www.planningalerts.org.au/api/howto#usage)
> 

with 

> 
Sign up for a 14-day free trial of our API to seamlessly integrate Planning Alerts data into your projects. Get your API key today and explore our services with a rate limit of about 1000 requests per day. [First read our terms of use](https://www.planningalerts.org.au/api/howto#usage)


Next, please **replace** 

>  Low volume, non–commercial use of the API service is free. We ask that personal and non–profit use of this service attribute the OpenAustralia Foundation on your website or application.

> 
> Standard agreements for commercial use are at [oaf.org.au/standard-agreements/planningalerts-commercial](https://www.oaf.org.au/standard-agreements/planningalerts-commercial/). Please [contact us](https://www.planningalerts.org.au/help/contact?reason=I+want+API+access+or+commercial+use) for commercial use. Commercial users may include Real Estate Agencies, Architects, Planners or Builders.
> 
> Also, please [get in touch](https://www.planningalerts.org.au/help/contact?reason=I+want+API+access+or+commercial+use) if you intend to use the service on a large scale. In order to maintain quality of service for our API users, this service is rate limited by default to approximately 1000 requests per day.
> 
> We offer a range of paid options, from rate–limited to unlimited use of this service.

with

###Apply for Ongoing Access
Enjoy uninterrupted access by signing up before your trial ends. Our [current standard agreement](https://www.oaf.org.au/standard-agreements/planningalerts-commercial/) applies for continued access to the service. 

If you have a clear community public purpose and are unfunded, you might qualify for a grant to access our API for free or at a reduced cost for community public purpose. Please [get in touch](https://www.planningalerts.org.au/help/contact?reason=I+want+API+access+or+commercial+use)

**Comment by katska** [comment]
so I think we can also ask the community purpose people to agree to terms and sign the agreement and make a different schedule either instead of or a variation on the payment. 

One question for me remains, whether we put additional terms for community public purpose access at the end of the agreement or in a separate place?

**Comment by katska** [comment]
@mlandauer 
It's important to make changes to set reasonable expectations of use for this service. 
If the new page suggested seems like too much change requiring decisions there's not headspace to make right now, please can you make the following changes in support of 
- introducing a trial
- holding users of the API accountable for their use
- bringing together 'terms of use' clearly in one place
The disadvantages in the short term are that people will need to get in touch with us after two weeks or so to continue to use the API, and that I'll need to manually monitor them.

Changes are as follow for a minimum viable next step:

1- remove this part

    Low volume, non–commercial use of the API service is free. We ask that personal and non–profit use
    
2- Replace with the following:

We require that all users 

3 - Remove "Low volume, non–commercial use of the API service is free. We ask that personal and non–profit use of this service attribute the OpenAustralia Foundation on your website or application. "

4- Remove "Also, please [get in touch](https://www.planningalerts.org.au/help/contact?reason=I+want+API+access+or+commercial+use) if you intend to use the service on a large scale. In order to maintain quality of service for our API users, this service is rate limited by default to approximately 1000 requests per day."

5 - Please [get in touch](https://www.planningalerts.org.au/help/contact?reason=I+want+API+access+or+commercial+use) for ongoing access to the API.

6 - Make **Usage** and **Be Nice** all one section. I can advise on switching the sentences around to flow, or have a go yourself.

**Comment by katska** [comment]
Turns out minimum viable doesnt require a two week trial, and with all the evolution above, now looks like this:

> 
> Get your API key today and explore our services with an initial rate limit of about 20 requests per day for evaluation purposes. If you need more, please [contact us](https://www.planningalerts.org.au/help/contact?reason=I+want+API+access+or+commercial+use)
> 
> Please [get in touch](https://www.planningalerts.org.au/help/contact?reason=I+want+API+access+or+commercial+use) for ongoing access to the API.
> 
> If you have a clear community public purpose and are unfunded, you might qualify for a grant to access our API for free or at a reduced cost. Please [ask us about this](https://www.planningalerts.org.au/help/contact?reason=I+want+API+access+or+commercial+use)
> 
> Attribution and a link to the Planning Alerts API are required on your website or application, or where ever you use the data.
> 
>

**Comment by katska** [comment]
@JoannaHill @mlandauer can we workshop what changes we can commit to on this page in the short term please?

**Comment by JoannaHill** [comment]
Yes. 

I can do a mock up of the text changes you're suggesting above. 
Where on the page would this go and should I be removing all the example calls?

**Comment by JoannaHill** [comment]
@katska You've made some other wording suggestions in Figma to this page. Lets regroup over there for fleshing out. Can Andy be given Figma access too?

https://www.figma.com/design/GEaOuajueHGvfaFPsOUZ9n?node-id=2663-325#600588808

**Comment by katska** [comment]
@JoannaHill  I find it quite hard to follow updates in figma and communicate a path when there are complex pieces as there are here, and the team is asynchronous and remote. And, I don't have admin access to figma so you or Matthew may need to add Andy.

**Comment by katska** [comment]
Let's move this to google docs and use the parts that feel useful based on where we're going now see [figma board](
https://www.figma.com/design/GEaOuajueHGvfaFPsOUZ9n?node-id=0-1#893997777).
When the bits are in google docs we can close this.

**Comment by katska** [comment]
@JoannaHill it looks like @blikantt still doesn't have figma access, and I don't have permissions so unable to help. 
There is a recent discussion here which is relevant to #1473

**Comment by JoannaHill** [comment]
Get the data section now called Planning Alerts API and includes major overhaul of information on the page. Including separting off Developer Documentation and making the landing page more of a sales page. See live site.
